src/components/common/DropDown/DropDown.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import "./DropDownStyles.css";
+
+class DropDown extends React.Component {
+    render() {
+        let dropDownElems = [];
+
+        for (let i = 0; i < this.props.contents.length; i++) {
```

Comment:
Research if this is the right way to use for loop or any other functionality code(I mean inside the render function). There are other ways like constructor and react Component life cycle etc. Learn about them and also learn when to use what.